README.md

```diff
@@ -1 +1,126 @@
-# hello-rails-react
\ No newline at end of file
+![](https://img.shields.io/badge/thecodechaser-blueviolet)
+
+# Hello Rails React App
+
+> Hello Rails React app is simple application that contains setup of rails and react in one application. The greeting page loads a random greeting from database on every page load.
+
+## Preview
+
+### Home Page
+
+![screenshot](./app/assets/images/home.png)
+
+### Greeting Page
+
+![screenshot](./app/assets/images/greeting.png)
+
+## Built With
+
+- Major languages (Ruby)
+- Framworks (Ruby on Rails)
+- Javascript (ReactJS)
+- Markup (HTML)
+- Styles (CSS, Tailwind)
+
+## Live version
+
+- Visit [Recipe-App]()
+
```

Comment:
- [ ] I would recommend removing this section if you don't have a live link to make the README professional.